### ISSUE TITLE
Safe Methods 

### ISSUE THREAD

**Issue by vairamuthushanmugavel** [javascript-tutorial/en.javascript.info#3356]
POST requests are shown as safe requests in the CORS section. MDN documentation, however, claims that it is a unsafe method.
MDN : https://developer.mozilla.org/en-US/docs/Glossary/Safe/HTTP

**Comment by joachimklug** [comment]
Agree.  Also RFC9110 states the same: https://httpwg.org/specs/rfc9110.html#safe.methods 
We could swap `POST` with `OPTIONS` and `TRACE` to complete the list.
